contracts/TokenConverter/RiskFundConverter.sol

```diff
@@ -239,6 +246,7 @@ contract RiskFundConverter is AbstractTokenConverter {
         address[] memory coreMarkets = IComptroller(corePoolComptroller).getAllMarkets();
 
         for (uint256 i; i < coreMarkets.length; ++i) {
+            if (vBNB == coreMarkets[i]) continue;
```

Comment:
I think that if `tokenAddress` is the address of WBNB we should return true in this function. The income generated in the vBNB market will reach the RiskFundConverter as WBNB (it's wrapped by the vBNBAdmin contract [here](https://github.com/VenusProtocol/venus-protocol/pull/289/files)). I would keep the check `vBNB == coreMarkets[i]` anyway, just in case the vBNB market is remove from the core pool in the future.

So, my proposal would be something like this:
```
if ((vBNB == coreMarkets[i] && tokenAddress == WBNB) || IVToken(coreMarkets[i]).underlying() == tokenAddress) {
	isAssetListed = true;
    break;	
}
```

wdyt?

Reply:
WBNB could be an immutable variable. We could name it NATIVE_WRAPPED, already considering deployment to different networks

Reply:
Added the checks, 1a41ef5c8209773aa381ce60479517a5f2253a69